spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "Food Index Page" do
+  it "displays list of attributes for food search results" do
+    food1 = Food.create(gtinupc: 124, description: "yummy food", brand_owner: "General Mills", ingredients: "hot dog, bun, mustard")
+    food2 = Food.create(gtinupc: 144, description: "yucky food", brand_owner: "General Mills", ingredients: "sweet potatoes")
```

Comment:
Rather than creating food in the database, what we're looking for here is you to just retrieve data from the API call. So instead of creating food for test data, you could use a tool like VCR to mock this API call